debian/marquee/01-certbot

```diff
@@ -2,6 +2,6 @@
 
 apt install certbot
 
-for domains in `cat DOMAINS`; do
-    certbot certonly -n --agree-tos --webroot -m admin@whatwg.org -d $domains -w /var/www/http
-done
+# https://stackoverflow.com/a/8714446
+DOMAIN_ARG="$(cat DOMAINS | awk -vORS=, '{ print }' | sed 's/,$//')"
+certbot certonly --agree-tos --webroot -m admin@whatwg.org -d "$DOMAIN_ARG" -w /var/www/http
```

Comment:
I removed `-n` because if the cert already exists one will be asked confirm extending it, and if `-n` is specified it just stops. There's a flag to extend without asking, but AFAICT that doesn't work if the cert wasn't already there. So this'll do, adding new domains isn't fully automated anyway.